assets/favicon/site.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Is this webmanifest used? I didn't see anything referencing it?

Reply:
I havent seen that too.

Reply:
Not anymore. The webmanifest was supposed to help with light/dark favicon for some browsers but at least in my testing it wasn't any more useful than not having it.